frontend/app/module/election/page/ElectionStatusPage.tsx

```diff
@@ -41,9 +191,59 @@ export function ElectionStatusPage() {
           </Button>
         </Alert>
       )}
-      <main>
-        <ElectionStatusProgress />
-        <article>Placeholder</article>
+      <main className={cls.statusMain}>
+        <div className={cls.statusTitle}>
+          <h2 id="status-title">{t("election_status.main_title")}</h2>
+          {/* TODO: Add button onClick to Create Polling Station page */}
+          <Button size="md" variant="secondary" leftIcon={<IconPlus />}>
+            {t("election_status.add_polling_station")}
+          </Button>
```

Comment:
It requires some page styling to properly add the button to the page, so that's why it's already implemented now. We can add the onClick once the #431 page/route exists.

Reply:
discussed, button stays, so resolved